### DISCUSSION TITLE
Invalid Plot after successful plotting

### DISCUSSION THREAD

**Discussion by Zurginator** [Chia-Network/chia-blockchain#6109] [Support]
Create a plot, copied it to an external HD:  (I could swear this only started occurring with 1.1.6)
Had 4 of these around the same time.

Failed to open (invalid plots)
These plots are invalid, you might want to delete them.

[plotter_log_1da52a0e-1bf1-4968-b7f3-ed8b9c9c06d3.txt](https://github.com/Chia-Network/chia-blockchain/files/6553752/plotter_log_1da52a0e-1bf1-4968-b7f3-ed8b9c9c06d3.txt)

**Comment by tiny0little** [comment]
logs looks fine. what do you see when you checking it?

```chia plots check -n 30 -g plot-k32-2021-05-24-21-45-d00f4```

**Reply by Zurginator** [reply]
Hope I did this right. I am in Windows so improvised.

[plotcheck.txt](https://github.com/Chia-Network/chia-blockchain/files/6554012/plotcheck.txt)


C:\Users\M1user\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon>chia plots check e:\plot-k32-2021-05-24-21-45-d00f46b4856d91b3d16554f916da13520e84c402da95c12e3e1c52ec08c96c25.plot
Usage: chia plots check [OPTIONS]
Try 'chia plots check -h' for help.

Error: Got unexpected extra argument (e:\plot-k32-2021-05-24-21-45-d00f46b4856d91b3d16554f916da13520e84c402da95c12e3e1c52ec08c96c25.plot)

**Reply by tiny0little** [reply]
it did not find it. is the plot still on drive e:\ ? if it is, try to use ```-g plot-k32-2021-05-24-21-45``` without full path or full name

**Reply by Zurginator** [reply]
Ok, I understand now. Actually found this same plot file on another HD and there it failed the check.
Src size incorrect. (accidentally closed the CMD window.... grrr)

Will cross check the rest of them and run plot check to see if anything else comes up.

Thanks!

**Reply by kevinwarnerlopez** [reply]
hola me paso lo mismo pudiste solucionar?

**Comment by BrandtH22** [comment]
@Zurginator , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
